tests/models/caffe2Models/predict_net.pbtxt

```diff
@@ -0,0 +1,25 @@
+name: "predict"
```

Comment:
You probably want to name this file differently, because future tests will add more predict_nets.

Reply:
What I meant here was renaming the file itself (e.g. `predict_net.pbtxt` -> `conv.pbtxt`), not changing name field of the proto. The name field is probably not important, because it's not used anywhere.